### DISCUSSION TITLE
Is this project dormant, dead or still on?

### DISCUSSION THREAD

**Discussion by jonasrichard** [TaKO8Ki/gobang#186] [General]
It would be nice if we can send pull requests and they are merged.

**Comment by EricDriussi** [comment]
was wondering the same thing. I cloned to project to try and implement some sort of configurable keymap just to help out with the TODOs only to find [this PR](https://github.com/TaKO8Ki/gobang/pull/153) just hanging there.
Any confirmation on the state of the project would be greatly appreciated, I would love to help with pending issues or PRs.

**Reply by jonasrichard** [reply]
Yeah, this is the problem with private repos if the owner gets too busy, and cannot focus on github. But inviting collaborators is always an option.
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository

**Reply by kyoto7250** [reply]
I am the person who posted the PR, and I am maintaining gobang in the repository below now.
https://github.com/kyoto7250/zhobo

If gobang is maintained again, I would like to continue contributing.

**Comment by NikodemMarek** [comment]
maybe it is possible to create a new main fork, and merge waiting prs into it, since @TaKO8Ki clearly is not available anymore?
if there is a big enough group of contributors it may be possible to bring this cool project back to life

**Comment by TrayserCassa** [comment]
I would fork it and maintain it but it's not golang :sweat_smile: I feel not to familiar with rust. But I would love if we give this a try, because scripting makes it very easy to get access quickly with 1000 databases.

**Reply by NikodemMarek** [reply]
I was willing to fork it myself, but i found another similar project https://github.com/achristmascarl/rainfrog, that seems to be actively developed
